utils/protobuf/datawritemetadata.proto

```diff
@@ -45,8 +45,8 @@ message IntegrationTimeMetadata {
         repeated uint32 pulse_position = 1;
       }
 
-      message PulsePhase {  // length = length of sequence
-        repeated float pulse_phase = 1;
+      message SequenceEncoding {
+        repeated float encoding_value = 1; // pulse x encoding length
```

Comment:
can I suggest the comment be fleshed out a bit more to indicate the dimensions possible

```suggestion
        repeated float encoding_value = 1; // num_pulses x encoding length (1 OR num_samps)
```